tester/json/consume_two_outputs.json

```diff
@@ -0,0 +1,39 @@
+{
+    "accounts": [
+        [
+            75000,
+            75000
+        ],
+        0
+    ],
+    "steps": [
+        {
+            "transaction": {
+                "from": 0,
```

Comment:
does this describe the sender ? therefore account 0 ?

Reply:
Yes